addons/rose/package.json

```diff
@@ -96,7 +96,7 @@
     "react-syntax-highlighter": "^15.5.0",
     "stylelint": "^15.10.1",
     "stylelint-config-prettier-scss": "^0.0.1",
-    "stylelint-config-standard-scss": "^6.0.0",
+    "stylelint-config-standard-scss": "^11.0.0",
```

Comment:
This change belongs to `rose` is this necessary within this PR? I think we want to keep the scope of the upgrade within Desktop?